src/components/SortDropdown.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  hiddenStyle: {
+    display: 'none',
+  },
+  selectedOptionTextStyle: {
+    fontWeight: '500',
+    fontFamily: 'Work Sans',
+    fontSize: '14px',
+    lineHeight: '18px',
+    margin: 'auto 0 auto 0',
```

Comment:
Can remove bottom and left values from margin property
```suggestion
    margin: 'auto 0',
```